src/binding/nats/mod.rs

```diff
@@ -0,0 +1,43 @@
+//! This module provides bindings between [cloudevents-sdk](https://docs.rs/cloudevents-sdk) and [nats](https://docs.rs/nats)
+//! ## Examples
+//! Deserialize [nats::Message](https://docs.rs/nats/latest/nats/struct.Message.html) into [Event](https://docs.rs/cloudevents-sdk/latest/cloudevents/event/struct.Event.html)
```

Comment:
is `latest` a good idea or should we point to an exact version here?

Reply:
Good point, I changed it to `0.21.0`